### ISSUE TITLE
automate parsing from upstream source

### ISSUE THREAD

**Issue by derhuerst** [hemanth/life-expectancy#4]
In order to keep this module up to date effortlessly, it is important to both link to the original source of the data and automate the build/cleanup/parsing process as much as possible.

**Comment by hemanth** [comment]
Yes, I have done the same with [inline-elements](https://github.com/hemanth/inline-elements) but the concern here is the data was fetched from an `xls` that was published, maybe we can write a simple script to scrape wikipedia tables (which has similar info) or you their API so that we need to do any scrapping?

**Comment by derhuerst** [comment]
> Yes, I have done the same with `inline-elements` […].

I can see that, in there, you generate `index.js`. I'd rather generate `inline-elements.json` and make `index.js` look like this:

```js
module.exports = requires('inline-elements.json')
```

Additionally, setting `main` to `inline-elements.json` might be the best choice.

**Comment by derhuerst** [comment]
> the concern here is the data was fetched from an xls that was published

[I parse `xlsx` automatically in the build script of `vbb-translate-ids`.](https://github.com/derhuerst/vbb-translate-ids/blob/8b8b7c0/build.js#L7) You may want to have a look.

**Comment by derhuerst** [comment]
Would you give me the link to the source file?

**Comment by hemanth** [comment]
I would still vote to play with the [API](https://en.wikipedia.org/w/api.php?action=parse&format=json&page=List_of_countries_by_life_expectancy&redirects&section=1)

**Comment by derhuerst** [comment]
Will ask in the Wikidata Slack channel.

**Comment by hemanth** [comment]
Any luck?

**Comment by derhuerst** [comment]
[Wikidata only has ~four~ three countries](https://query.wikidata.org/#SELECT%20%3Fcountry%20%3FcountryLabel%20%3FlifeExpectancy%20WHERE%20%7B%0A%20%20%3Fcountry%20wdt%3AP31%20wd%3AQ3624078%3B%0A%20%20%20%20%20%20%20%20%20%20%20wdt%3AP2250%20%3FlifeExpectancy.%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22de%22.%20%7D%0A%7D).

**Comment by hemanth** [comment]
[Look](https://en.wikipedia.org/wiki/List_of_countries_by_life_expectancy)!

**Comment by derhuerst** [comment]
Yes, this is the Wiki article. Its data is not stored in Wikidata unfortunately (which is how it's supposed to be). We can still automatically parse that one, e.g. using [cheerio](https://github.com/cheeriojs/cheerio#readme).
